src/Razor/src/Microsoft.AspNetCore.Razor.Language/DefaultRazorParserOptions.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.AspNetCore.Razor.Language
 {
     internal class DefaultRazorParserOptions : RazorParserOptions
     {
-        public DefaultRazorParserOptions(DirectiveDescriptor[] directives, bool designTime, bool parseLeadingDirectives, RazorLanguageVersion version)
+        public DefaultRazorParserOptions(DirectiveDescriptor[] directives, bool designTime, bool parseLeadingDirectives, RazorLanguageVersion version, string fileKind)
```

Comment:
😻 

At least I had the idea that we should do this.

Reply:
Talked offline, `FileKind` shouldn't be part of the parser options since 1 project engine can parse both Components and Legacy Views.  Implications of removing this means that we'll need to change the consumers to rely on the `CodeDocument` or other per-file information to more accurately represent when a parse is operating on a Component document vs. a Legacy document.